src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -276,14 +276,17 @@ private void configureBindings() {
         .onFalse(shortRumble(m_driveController.getHID(), RumbleType.kBothRumble));
 
     // Rumble controllers when target is detected and we don't have a note
-    NetworkTableEntry limelightTargetDetected =
-        NetworkTableInstance.getDefault().getTable("limelight").getEntry("tv");
-    new Trigger(() -> limelightTargetDetected.getDouble(0.0) == 1.0)
+    NetworkTableEntry photonvisionTargetDetected =
+        NetworkTableInstance.getDefault().getTable("photonvision/Fisheye").getEntry("targetYaw");
```

Comment:
`hasTarget` is the correct nt key
https://docs.photonvision.org/en/latest/docs/additional-resources/nt-api.html#getting-target-information

it is a boolean